files/default/neutron-ha-tool.py

```diff
@@ -171,22 +174,25 @@ def run(args):
     if not auth_version:
         auth_version = os.getenv('OS_IDENTITY_API_VERSION', None)
         if not auth_version:
-            auth_version = 3
+            auth_version = '2.0'
```

Comment:
I don't understand why we've cherry-picked a commit to add v3 support, and then immediately regress to v2.0?  If this is really correct then the commit message needs to explain why.